framework/include/meshmodifiers/ElementSubdomainModifierBase.h

```diff
@@ -12,6 +12,16 @@
 #include "ElementUserObject.h"
 #include "NonlinearSystemBase.h"
 #include "AuxiliarySystem.h"
+#include "NodalPatchRecoveryBase.h"
+#include "KDTree.h"
+
+enum class ICStrategy
```

Comment:
Remove the IC_ prefix in the enum

Reply:
What's the difference between IC_POLYNOMIAL and IC_POLYNOMIAL_WHOLE_SOLVED_DOMAIN?